app/components/obs/inputs/ObsInput.ts

```diff
@@ -8,31 +8,31 @@ import {
   isNumberProperty,
   isTextProperty,
   isFontProperty,
-  isPathProperty
+  isPathProperty,
 } from '../../../util/properties-type-guards';
 import { $t } from 'services/i18n/index';
 
 /**
  * all possible OBS properties types
  */
 export declare type TObsType =
-  'OBS_PROPERTY_BOOL' |
-  'OBS_PROPERTY_INT' |
-  'OBS_PROPERTY_LIST' |
-  'OBS_PROPERTY_PATH' |
-  'OBS_PROPERTY_FILE' |
-  'OBS_PROPERTY_EDIT_TEXT' |
-  'OBS_PROPERTY_TEXT' |
-  'OBS_PROPERTY_UINT' |
-  'OBS_PROPERTY_COLOR' |
-  'OBS_PROPERTY_DOUBLE' |
-  'OBS_PROPERTY_FLOAT' |
-  'OBS_PROPERTY_SLIDER' |
-  'OBS_PROPERTY_FONT' |
-  'OBS_PROPERTY_EDITABLE_LIST' |
-  'OBS_PROPERTY_BUTTON' |
-  'OBS_PROPERTY_BITMASK' |
-  'OBS_INPUT_RESOLUTION_LIST';
+  | 'OBS_PROPERTY_BOOL'
```

Comment:
TypeScript is actually technically fine it.  I think the idea here is that every line is formatted the same way, and it's easy to insert anywhere in the list, similar to the trailing comma rule.  I also somewhat cynically believe most people like this because they like the straight line all the `|`s make on the left.